### PR TITLE
Add error to unsucessful api call on game details

### PR DESCRIPTION
**Pull request by aspitz1**
## What does this PR do?
- [ ] New feature (adds functionality)
- [x] Debugging/Refactor
 
### Where should your reviewer start?
- GameDetailScreen
### What (if any) features are you implementing?
- Adding text to screen that gives error feedback to user
### What (if anything) did you refactor?
- Split add/delete functions into their own functions vs one with an if statement
### What issues does this PR close?
- #70 
### Is there anything that you need from your teammates?
- I tested it by intentionally commenting out a part of the url
### Any other comments, questions, or concerns?
- n/a
### How Has This Been Tested?
- [x] I tested my changes in the browser
- [ ] I tested with Cypress

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new bugs
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by tfettig22 — APPROVED** [review]
Pulled down and tested, seems to be working great! Thanks for adding this in

File: src/apiCalls.js [hunk 1]
```diff
@@ -73,7 +73,11 @@ const deleteGame = (userID, gameID) => {
         "Content-Type": "application/json",
       },
     }
-  )
+  ).then(response => {
+    if(!response.ok) {
+      throw new Error("Looks like something looked wrong.")
```
Comment (NickLiggett) [inline src/apiCalls.js:88]: The English student in me wonders if we can come up with a better phrase. Too many "looks". What a funny word... look, look, look....

File: src/components/GameDetailsScreen.js [hunk 2]
```diff
@@ -28,34 +28,43 @@ const GameDetailsScreen = ({
...
+
+    deleteGame(userID, foundGame.id)
+      .then(() => {
+        removeGame(game.id);
+        setHasGame(false);
+        setError("");
+      })
+      .catch((error) => {
+        console.log(error);
+        setError("Looks like something went wrong.");
```
Comment (NickLiggett) [inline src/components/GameDetailsScreen.js:48]: Great job error handling! Thank you, Anna!

**Review by NickLiggett — APPROVED** [review]
Thanks Anna! Great work!

**Pull request merged**